package.json

```diff
@@ -14,7 +14,7 @@
     "dev:test": "jest --watch",
     "build": "tsc",
     "test": "jest",
-    "clean": "rm -rf dist && git clean -fx",
```

Comment:
I screwed myself pretty bad with this. I lost a great test suite, hour+ of work. Very luckily I had the implementation open in VSCode and it was kept around (marked as deleted). I recovered it. Feel pretty stupid :D